js-intro.js

```diff
@@ -37,7 +62,35 @@ var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
 
+const noVowels1 = str => {
+
+    //turn string into an array
+
+    var array1 = stringWithVowels1.split('')
```

Comment:
Your logic is great here, but on this line, you are hard-coding a specific variable into your function. This doesn't allow this function to be reusable. Rather than saying `var array1 = stringWithVowels1.split('')`, consider utilizing the argument you've set up in the line above, so that _any_ array can be passed to this function.